industrial_ci/src/env.sh

```diff
@@ -16,6 +16,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if [ ! "$CATKIN_CONFIG" ]; then export CATKIN_CONFIG="--install"; fi
```

Comment:
https://github.com/ros-industrial/industrial_ci/pull/177#issuecomment-301912881
> I want to enforce --install as long as --no-install not present.

Default value is exported here, so yes `CATKIN_CONFIG=--install` is enforced IMO. Is this not sufficient / not the way it should be?